recipe/meta.yaml

```diff
@@ -1,19 +1,20 @@
 {% set name = "synphot" %}
-{% set version = "0.1.3" %}
+{% set version = "0.2.0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: c512ecd3dd2fb46830c8e99036f14e5bf03e8b624bd3c931e46676ff0a78b304
+  sha256: e127fa4ac98fe5cb1335b76c0de18fef4159849a302f03517f7c83c530cb92e9
   patches:
     - setup.cfg.patch
 
 build:
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py2k]
```

Comment:
@mwcraig , is this not how to skip PY2 builds? The CI still runs PY2 tests.

Reply:
It is better to add a version restriction to python in the host/run requirements, e.g. `python >=3.6` or whatever the requirement is. After that you may need to rerender by copy/pasting `@conda-forge-admin please rerender` into a comment.

Reply:
Hmmm, looks like the bot re-rendered even though that is backticks. 🤷‍♂️

Looks like it dropped python 2 in any event...

Reply:
So, is this PR good as-is? Or should I still change the Python incantation?